apps/dashboard/src/components/subscribers/subscriber-tabs.tsx

```diff
@@ -0,0 +1,73 @@
+import { RiCloseLine, RiGroup2Line } from 'react-icons/ri';
+import { useNavigate } from 'react-router-dom';
+import { CompactButton } from '../primitives/button-compact';
+import { Separator } from '../primitives/separator';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '../primitives/tabs';
+import SubscriberOverviewForm from './subscriber-overview-form';
+import useFetchSubscriber from '@/hooks/use-fetch-subscriber';
+import TruncatedText from '../truncated-text';
+
+const tabTriggerClasses =
+  ' hover:data-[state=inactive]:text-foreground-950 h-11 data-[state=active]:border-b data-[state=active]:border-primary-base data-[state=active]:border-b-2 py-3 rounded-none [&>span]:h-5 px-0';
+
+export default function SubscriberTabs({ subscriberId }: { subscriberId: string }) {
+  const navigate = useNavigate();
+  const { data, isFetching } = useFetchSubscriber({ subscriberId });
```

Comment:
I'm wondering if it would be better to call `useFetchSubscriber()` directly in the tab form since I assume the data would be used only there. 

Because e.g. subscriber preferences is another endpoint, if I would to  follow same pattern I would add it also here, then in the end all 4 API calls for each tab would be executed upon opening the drawer, even thought the user might not visit the tab.

Reply:
agree, will move it to the overview tab